client/src/js/components/bhEntitySelect/bhEntitySelect.tmpl.html

```diff
@@ -0,0 +1,32 @@
+<div ng-form="EntityForm" bh-entity-select ng-model-options="{ updateOn: 'default' }">
+  <div
+    class="form-group"
+    ng-class="{ 'has-error' : $ctrl.validateTrigger && EntityForm.entity_uuid.$invalid }">
+
+    <label ng-hide="$ctrl.label" class="control-label" translate>
+      ENTITY.LABEL
+    </label>
+
+    <label ng-show="$ctrl.label" class="control-label" translate>
+      $ctrl.label
+    </label>
+    
+    <ng-transclude></ng-transclude>
+    <ui-select 
+      name="entity_uuid" 
+      ng-model="$ctrl.entityUuid"
+      on-select="$ctrl.onSelect($item, $model)"
+      ng-required="$ctrl.required">
+      <ui-select-match placeholder="{{ 'FORM.SELECT.ENTITY' | translate }}">
+        <span>{{$select.selected.display_name}}</span>
+      </ui-select-match>
+      <ui-select-choices ui-select-focus-patch repeat="entity.uuid as entity in $ctrl.entities | filter: { 'display_name': $select.search }">
+        <span ng-bind-html="entity.display_name | translate | highlight:$select.search"></span>
```

Comment:
If you need this translated, it is better to do it in JavaScript for the following reasons:
 1. It is slightly faster.
 2. Your `filter: { 'display_name' : $select.search }` logic will work correctly.  Currently, it operates on the translation _key_ not the translated _value_.